.github/workflows/test_profiles.yml

```diff
@@ -65,6 +70,8 @@ jobs:
           SF_DATABASE: ${{ secrets.SF_DATABASE }}
           SF_WAREHOUSE: ${{ secrets.SF_WAREHOUSE }}
           SF_SCHEMA: ${{ secrets.SF_SCHEMA }}
+          # Duckdb
+          DB_PATH: ${{ github.workspace }}/temp/duck.db
```

Comment:
if we remove the `/temp` part we can remove the `mkdir temp` step right? Can we do that?

Reply:
yea simpler